plugin.video.embycon/resources/lib/widgets.py

```diff
@@ -170,61 +170,69 @@ def getWidgetContentSimilar(handle, params):
+        series_id = result.get("SeriesId")
+        if series_id:
+            params["id"] = series_id
+            return get_widget_content_cast(handle, params)
+
+    list_items = []
     if result is not None:
-        people = result.get("People")
+        people = result.get("People", [])
     else:
-        people = None
-
-    if people is not None:
-        for person in people:
-            #if (person.get("Type") == "Director"):
-            #    director = director + person.get("Name") + ' '
-            #if (person.get("Type") == "Writing"):
-            #    writer = person.get("Name")
-            #if (person.get("Type") == "Writer"):
-            #    writer = person.get("Name")
-            if (person.get("Type") == "Actor"):
-                person_name = person.get("Name")
-                person_role = person.get("Role")
-                person_id = person.get("Id")
-                person_tag = person.get("PrimaryImageTag")
-                person_thumbnail = None
-                if person_tag:
-                    person_thumbnail = downloadUtils.imageUrl(person_id, "Primary", 0, 400, 400, person_tag, server=server)
-
-                if kodi_version > 17:
-                    list_item = xbmcgui.ListItem(label=person_name, iconImage=person_thumbnail, offscreen=True)
-                else:
-                    list_item = xbmcgui.ListItem(label=person_name, iconImage=person_thumbnail)
-
-                if person_thumbnail:
-                    artLinks = {}
-                    artLinks["thumb"] = person_thumbnail
-                    artLinks["poster"] = person_thumbnail
-                    list_item.setArt(artLinks)
-
-                labels = {}
-                labels["mediatype"] = "artist"
-                list_item.setInfo(type="music", infoLabels=labels)
-
-                if person_role:
-                    list_item.setLabel2(person_role)
-
-                itemTupple = ("", list_item, False)
-                listItems.append(itemTupple)
+        people = []
+
+    for person in people:
+        # if (person.get("Type") == "Director"):
+        #     director = director + person.get("Name") + ' '
+        # if (person.get("Type") == "Writing"):
+        #     writer = person.get("Name")
+        # if (person.get("Type") == "Writer"):
+        #    writer = person.get("Name")
+        if person.get("Type") == "Actor":
+            person_name = person.get("Name")
+            person_role = person.get("Role")
+            person_id = person.get("Id")
+            person_tag = person.get("PrimaryImageTag")
+            person_thumbnail = None
+            if person_tag:
+                person_thumbnail = downloadUtils.imageUrl(person_id, "Primary", 0, 400, 400, person_tag, server=server)
+
+            if kodi_version > 17:
+                list_item = xbmcgui.ListItem(label=person_name, iconImage=person_thumbnail, offscreen=True)
```

Comment:
please don't use `iconImage` kwarg it is deprecated. Set the icon with `list_item.setArt` instead